src/data/hassio.ts

```diff
@@ -225,3 +240,85 @@ export const fetchHassioSnapshotInfo = (
       `hassio/snapshots/${snapshot}/info`
     )
     .then(hassioApiResultExtractor);
+
+export const setHassioAddonOption = async (
+  hass: HomeAssistant,
+  slug: string,
+  data: HassioAddonSetOptionParams
+) => {
+  await hass.callApi<HassioResponse<void>>(
+    "POST",
+    `hassio/addons/${slug}/options`,
+    data
+  );
+};
+
+export const setHassioAddonSecurity = async (
+  hass: HomeAssistant,
+  slug: string,
+  data: HassioAddonSetSecurityParams
+) => {
+  await hass.callApi<HassioResponse<void>>(
+    "POST",
+    `hassio/addons/${slug}/security`,
+    data
+  );
+};
+
+export const installHassioAddon = async (hass: HomeAssistant, slug: string) => {
+  await hass.callApi<HassioResponse<void>>(
+    "POST",
+    `hassio/addons/${slug}/install`
+  );
+};
+
+export const uninstallHassioAddon = async (
+  hass: HomeAssistant,
+  slug: string
+) => {
+  await hass.callApi<HassioResponse<void>>(
+    "POST",
+    `hassio/addons/${slug}/uninstall`
+  );
+};
+
+export const fetchHassioHardwareAudio = async (hass: HomeAssistant) => {
+  const response = await hass.callApi<HassioResponse<HassioHardwareAudioList>>(
+    "GET",
+    "hassio/hardware/audio"
+  );
+  return response.data;
```

Comment:
Why not use `hassioApiResultExtractor` like above?

Reply:
https://github.com/home-assistant/home-assistant-polymer/pull/4398#discussion_r362767753